### DISCUSSION TITLE
/guide/update.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#13] [Announcements]
# /guide/update.html

VanBlog 的官方网站

https://vanblog.mereith.com/guide/update.html

**Comment by yanhaoxue** [comment]
没关闭服务，直接运行了一键升级脚本，然后就前端查看文章时一直显示500Internal Server Error.卸载重装后还是一样的，缓存清理后也不行完整的日志是这个

vanblog_1  | [Nest] 19  - 02/08/2023, 12:31:22 AM   ERROR [WebsiteProvider] TypeError: Cannot read properties of undefined (reading 'siteLogo')
vanblog_1  |     at getLayoutProps (/app/website/.next/server/chunks/47.js:2454:28)
vanblog_1  |     at getPostPagesProps (/app/website/.next/server/chunks/47.js:2717:25)
vanblog_1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
vanblog_1  |     at async getStaticProps (/app/website/.next/server/pages/post/[id].js:173:16)
vanblog_1  |     at async Object.renderToHTML (/app/website/node_modules/next/dist/server/render.js:385:20)
vanblog_1  |     at async doRender (/app/website/node_modules/next/dist/server/base-server.js:720:34)
vanblog_1  |     at async cacheEntry.responseCache.get.incrementalCache.incrementalCache (/app/website/node_modules/next/dist/server/base-server.js:837:28)
vanblog_1  |     at async /app/website/node_modules/next/dist/server/response-cache/index.js:83:36

**Reply by Mereithhh** [reply]
抱歉，这个是 0.45.4 的问题，我正在紧急修复。
